pkgs/games/osu-lazer/default.nix

```diff
@@ -59,6 +63,9 @@ buildDotnetModule rec {
   fixupPhase = ''
     runHook preFixup
 
+    wrapProgram $out/bin/osu\! \
```

Comment:
Does this work instead?
```suggestion
    wrapProgram "$out/bin/osu!" \
```

Reply:
Yep, that works too. Do you think we should use this everywhere? There are a lot of `\!`s.

Reply:
I would prefer to avoid escaping `!` since it makes harder to read, so yes, if you can replace every occurence it would be useful.

Reply:
After this change I confirmed the desktop files pass `desktop-file-validate` checks and they can be launched with `gtk-launch`.